modules/registry-operator-configuration-resource-overview.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * assembly/registry
+// * openshift_images/configuring-registry-operator.adoc
+
+
+[id="registry-operator-configuration-resource-overview-{context}"]
+= Image Registry Operator configuration
+
+The image-registry resource offers the following configuration fields.
```

Comment:
\`image-registry\` ?
Are these UI fields or parameters in the yaml definition? If they're in the yaml, they should be "configuration parameters."